### PR TITLE
Update gain_models.py

### PR DESCRIPTION
**Pull request by GiovanniVolta**
Gas xenon gain during cooling phase: 2020-08-26. PTMs at 1.5 kV a part PMT 116 and 195 which were at 1,3 kV. PMTs 354, 362, 386, 427 were off.

Note that during led calibration of August 26th PMTs 264, 272, 285, 300, 308, 311, 313, 316, 325, 339, 356, 359 didn't acquire data. For them I have used the results from 2020-08-23. 
For PMT 137, which were always off, I have used the result from 2020-07-24.

### CODE REVIEW

**Comment by WenzDaniel** [comment]
Hi Giovanni, thanks. @feigaodm @skazama  this change is ready for merge.

**Review by WenzDaniel — APPROVED** [review]

**Comment by skazama** [comment]
@GiovanniVolta @WenzDaniel Thanks. Do we also need to change [here](https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L25) until a nice CMT tool becomes ready to handle?

**Comment by WenzDaniel** [comment]
If these cane shall be taken as the new defaults yes.

**Comment by skazama** [comment]
I think the model which @GiovanniVolta implemented can be used as a default until we have CMT tool to handle the time-dependent model. @GiovanniVolta Can you please also modify [https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L25](here)?

**Review by jorana — CHANGES_REQUESTED** [review]
Perhaps it's good to mention that if we indeed use this model we have to reprocess the following plugins (given that we implement @skazama comment):
afbeelding

Furthermore, we could use this repo https://github.com/XENONnT/strax_auxiliary_files/ to start saving these gains.

**Comment by GiovanniVolta** [comment]
> 
> 
> I think the model which @GiovanniVolta implemented can be used as a default until we have CMT tool to handle the time-dependent model. @GiovanniVolta Can you please also modify [https://github.com/XENONnT/straxen/blob/master/straxen/contexts.py#L25](here)?

Sorry Shingo, I have been busy today. I will do it

**Comment by GiovanniVolta** [comment]
I have added in the context the new gain model:  'TemporaryGXe_1500V_PMT116_1300_PMT195_1300'

File: straxen/contexts.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     n_nveto_pmts=120,
     n_tpc_pmts=straxen.n_tpc_pmts,
     n_top_pmts=straxen.n_top_pmts,
-    gain_model=('to_pe_constant', '1300V_20200428'),
+    gain_model=('to_pe_constant', '1300V_20200428', 'TemporaryGXe_1500V_PMT116_1300_PMT195_1300'),
```
Comment (jorana) [inline straxen/contexts.py:25]: Hi Giovanni, thanks! Can you change it to:
```
gain_model=('to_pe_constant','TemporaryGXe_1500V_PMT116_1300_PMT195_1300'),
```

**Review by jorana — CHANGES_REQUESTED** [review]

**Review by jorana — APPROVED** [review]

**Pull request merged**